ngraph/core/src/op/abs.cpp

```diff
@@ -81,6 +73,9 @@ namespace absop
 
 bool op::Abs::evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const
 {
-    OV_ITT_SCOPED_TASK(itt::domains::nGraphOp, "op::Abs::evaluate");
-    return absop::evaluate_abs(inputs[0], outputs[0], shape_size(get_output_shape(0)));
+    bool rc = false;
+    NGRAPH_OP_SCOPE(
+        Abs_evaluate,
```

Comment:
Maybe it is better to use `v0_Abs_evaluate` name here to be consistent?

Reply:
Fixed